website_cookie_notice/templates/website.xml

```diff
@@ -0,0 +1,45 @@
+<odoo>
+    <template
+        id="loader"
+        inherit_id="website.assets_frontend"
+        name="Cookies script loader"
+    >
+        <xpath expr=".">
+            <script
+                type="text/javascript"
+                src="/website_cookie_notice/static/src/js/accept_cookies.js"
+            />
+        </xpath>
+    </template>
+    <template id="message" name="Cookies notice">
+        <t t-set="cookie_legal_page" t-value="website.cookie_notice_legal_page_id" />
+        <div
+            t-if="request and
+        not request.session.get('accepted_cookies')"
+            id="website_cookie_notice"
+            class="container-fluid cc-cookies alert alert-warning"
+            role="alert"
+        >
+            <div class="container">
+                <div class="row">
+                    <div
+                        class="col-sm-10"
+                    >We use cookies in this website. Read about them in our <a
+                            t-att-href="cookie_legal_page and cookie_legal_page.url or '#'"
```

Comment:
As explained above, do not use logic, just a raw link to `/shop/terms#privacy` (which most likely will be the default privacy link as discussed in #670).